GitLink.py

```diff
@@ -48,86 +42,70 @@ def run(self, edit, **args):
         # Switch to cwd of file
         os.chdir(path + "/")
 
-        # Find the repo
-        git_config_path = self.getoutput("git remote show origin -n")
-
-        # Determine git URL which may be either HTTPS or SSH form
-        # (i.e. https://domain/user/repo or git@domain:user/repo)
-        #
-        # parts[0][2] will contain 'domain/user/repo' or 'domain:user/repo'
-        #
-        # see https://regex101.com/r/pZ3tN3/2 & https://regex101.com/r/iS5tQ4/2
-        p = re.compile(r"(.+: )*([\w\d\.]+)[:|@]/?/?(.*)")
-        parts = p.findall(git_config_path)
-        git_config = parts[0][2]
-
-        remote_name = 'github'
-        if 'bitbucket' in git_config:
-            remote_name = 'bitbucket'
-        if 'codebasehq.com' in git_config:
-            remote_name = 'codebasehq'
-        if 'gitlab' in git_config:
-            remote_name = 'gitlab'
-        remote = REMOTE_CONFIG[remote_name]
-
-
-        # need to get username from full url
-
-        # Get username and repository (& also project for codebasehq)
-        remote_without_git = re.sub('.git$', '', git_config)
-        if ':' in git_config:
-            # SSH repository
-            if remote_name == 'codebasehq':
```

Comment:
Same here, as I'm not completely familiar with when that PR merged, but I'm sure removing this will break the project for some subset of people.